source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.h

```diff
@@ -63,3 +64,5 @@ void TextBlockToRichTextElementProperties(const std::shared_ptr<TextBlock> &text
 void TextRunToRichTextElementProperties(const std::shared_ptr<TextRun> &textRun, RichTextElementProperties &textProp);
 
 void buildIntermediateResultForText(ACRView *rootView, ACOHostConfig *hostConfig, RichTextElementProperties const &textProperties, NSString *elementId);
+
+ACOBaseActionElement *deserializeUnknowActionToCustomAction(const std::shared_ptr<UnknownAction> action);
```

Comment:
>Unknow [](start = 33, length = 6)

Unknown

Reply:
thanks

---
In reply to: [338169519](https://github.com/microsoft/AdaptiveCards/pull/3526#discussion_r338169519) [](ancestors = 338169519)